src/components/CheckboxIOS.tsx

```diff
@@ -0,0 +1,69 @@
+// @ts-nocheck
+import * as React from "react";
+import { View } from "react-native";
+import Icon from "./Icon";
+import Touchable from "./Touchable";
+import { withTheme } from "../core/theming";
+import themeI from "../styles/DefaultTheme";
+
+interface Props extends TouchableHighlightProps {
+  status: "checked" | "indeterminate" | "unchecked";
+  disabled: boolean;
+  onPress: () => void;
+  theme: typeof themeI;
+  color?: string;
+}
+
+const CheckboxIOS: React.FC<Props> = ({
+  status,
+  disabled,
+  onPress,
+  theme,
+  color,
+  ...rest
+}) => {
+  //const displayName = "Checkbox.IOS";
```

Comment:
mind removing this? or uncommenting if it works?

Reply:
it was there in the original component but was never used anywhere. Will remove it